### PR TITLE
added apk_size check using python script (fix #692)

### LINKED ISSUE #692: APK size script fails on buddybuild
```
2852    Checking release APK file sizes ...
2853    stat: invalid option -- '%'
2854    Try 'stat --help' for more information.
2855    tools/metrics/apk_size.sh: line 18: let: filesize=: syntax error: operand expected (error token is "=")
2856     * [OKAY] app-focus-webkit-release.apk ( <= 4194304)
2857    stat: invalid option -- '%'
2858    Try 'stat --help' for more information.
2859    tools/metrics/apk_size.sh: line 18: let: filesize=: syntax error: operand expected (error token is "=")
2860     * [OKAY] app-klar-webkit-release.apk ( <= 4194304)
```

### PR DESCRIPTION
**Pull request by mastizada**

### CODE REVIEW

File: tools/metrics/apk_size.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+from os import path, listdir, stat
+from sys import exit
+
+SIZE_LIMIT = 4194304
+PATH = path.join(path.dirname(path.abspath(__file__)), '../../app/build/outputs/apk/')
+
+files = []
+import code
+code.interact(local=locals())
```
Comment (pocmo) [inline tools/metrics/apk_size.py]: What is this for?

File: tools/metrics/apk_size.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+try:
+    files = [f for f in listdir(PATH) if path.isfile(path.join(PATH, f)) and f.endswith('.apk') and "release" in f]
+except OSError as e:
+    if e.errno == 2:
+        print("Directory is missing, build apk first!")
+        exit(1)
+    print("Unknown error while reading list of files")
```
Comment (pocmo) [inline tools/metrics/apk_size.py]: Can we print the exception/error here?

File: tools/metrics/apk_size.py (hunk 1 above)
Comment (mastizada) [inline tools/metrics/apk_size.py]: sorry, that was for debugging, removed.

File: tools/metrics/apk_size.py (hunk 2 above)
Comment (mastizada) [inline tools/metrics/apk_size.py]: yes, makes sense, added

**Pull request merged**